### PR TITLE
Add mock command fixture

### PR DESCRIPTION
**Pull request by MartinHjelmare**

### CODE REVIEW

File: test/conftest.py [hunk 1]
```diff
@@ -59,6 +60,41 @@ async def client_fixture(loop, client_session, ws_client, uuid4):
...
+    async def set_response(message):
+        """Check the message and set the mocked response if a command matches."""
+        for match_command, response in mock_responses:
+            if all(message[key] == value for key, value in match_command.items()):
+                ack_commands.append(message)
+                received_message = {
+                    "type": "result",
+                    "messageId": uuid4,
+                    "result": response,
+                    "success": response["success"],
```
Comment (Michsior14) [inline test/conftest.py:86]: Not every response will have `success`.

**Comment by Michsior14** [comment]
While I don't have any preferred way how to maintain the tests, this seems a bit cleaner to me.

File: test/conftest.py (hunk 1 above)
Comment (MartinHjelmare) [reply test/conftest.py:86]: The result messages should either have success True or success False, right?

Comment (MartinHjelmare) [reply test/conftest.py:86]: I guess we're not limited to test result messages with this fixture.

Comment (MartinHjelmare) [reply test/conftest.py:86]: I'll fix this.

File: test/conftest.py [hunk 2]
```diff
@@ -59,6 +60,41 @@ async def client_fixture(loop, client_session, ws_client, uuid4):
...
+                }
+                client.async_handle_message(received_message)
```
Comment (balloob) [inline test/conftest.py:88]: If you really want to mock the transport, you would not call this method directly but instead have this message be set as result for the open `msg = await client.receive()` 😉

Comment (balloob) [reply test/conftest.py:88]: (it's fine like this)

File: test/conftest.py (hunk 1 above)
Comment (balloob) [reply test/conftest.py:86]: Every command has a result right now. So this is fine.

**Review by balloob — APPROVED** [review]

**Pull request merged**